coding-concepts.js

```diff
@@ -8,36 +8,36 @@
 // --------------------1) What will this log?
 
 const cohort = "Bravo 2023"
-// console.log(cohort.split(""))
+console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: the .split method takes a string and turns it into an array. It will take all the letters inside of the string and put them into their own index each. 
+// b) Verify and explain: I was correct. Since we said .split(""), it took each character and put it into its own index inside an array.
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
   `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: The code will not return anything because the function provided the cardinal rule of functions; which is to always include a return. In this case, since we are not telling the function to return the string, it will not output anything
+// b) Verify and explain: This was correct. The console logged undefined. This is because since we did not tell it to return anything, it recognizes it as undefined.
 
 // --------------------3) What will this log?
 
 const multipliedByTwo = [4, 5, 6, 7, 8].map((number) => number * 2)
-// console.log(multipliedByTwo)
+console.log(multipliedByTwo)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: it will not return anything because it's missing a return
+// b) Verify and explain: it returned a new array with the numbers inside of the array multiplied by 2. I see it does not need a return because it is not a function.
 
```

Comment:
It does not need a return because there aren't additional lines in which case we would need {} and a return.